gramps/plugins/webreport/narrativeweb.py

```diff
@@ -1940,9 +1940,9 @@ def __add_report_options(self, menu):
         category_name = _("Report Options")
         addopt = partial(menu.add_option, category_name)
 
-        self.__archive = BooleanOption(_('Store web pages in .tar.gz archive'),
+        self.__archive = BooleanOption(_('Store webpages in .tar.gz archive'),
```

Comment:
Whilst the use of "website" seems to be more common than "web site", I'm not too sure about " webpages".  I think that it would be better the use "website" here.